### PR TITLE
Hexagons moved to own component

### PR DESCRIPTION
**Pull request by rschloen**
Created new component, Hexagons, to hold actuator layout so that it can be used more easily in other components(Manual, Automatic). 

At the moment, Hexagon component dispatches an event when an actuator is clicked which calls functions in Manual to generate the actuator commands. This will likely be changed to make actuator control more global rather than repeating the functions that generate the command in every parent component. (Note: Automatic component does not currently have functions that build commands in anticipation of new state tracking scheme).

Created new orientation for the hexagon layout: vertical. Orientation refers to the direction in which the hexagons are in line with their neighbors. For example, actuators 0-3 line up in a column in the vertical orientation, and a row in horizontal.

### CODE REVIEW

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]: In the future, I would like to take a hexagon config as a props and some intermediate components would take in orientation or hexagon config ids and spit out the horizontalHexagons array and transformation string

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]: spacing is weird here

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte:75]: This is the bit that should be computed to a degree to avoid clipping depending on where things are placed. Let's worry about fixing this once we have more stuff in place. We will have to account for it if we are going to overlay the  fabric on the body placement images in the scrollable options. ... Maybe we can skip handling the spacing math and get the svg scaling to handle it for us in places where we don't set touch input handlers.

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]: I don't like how all of this is duplicated. This seems like it will be hard to translate to stichable fabrics. What is your plan?

**Review by trueb2 — CHANGES_REQUESTED** [review]
From some of the stuff that Yei Hwan showed me, I think we need to use translation math to turn acutator blocks instead of if statements. If we go diagonal, then this gets very hard to check.

If we add the math for rotation to the hit check to the handleTouchMove, then we only need to change the rotation on the svg. All of the ids, padding, and centers will still be correct

File: src/routes/Manual.svelte
Comment (trueb2) [inline src/routes/Manual.svelte:272]: 🤔

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]: style-wise, I would put exports (external immutable globals), then lets (mutable globals), then consts (immutable globals).

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]:
```suggestion
                    transform={`translate(${hexagon.x - (hexagon.width / 2)} ${hexagon.y - (hexagon.height / 2)})`}
```

File: src/Utils/Hexagons.svelte
Comment (trueb2) [inline src/Utils/Hexagons.svelte]: This log was probably useful for debugging but won't be in the future. I would recommend deleting.

File: src/routes/Automatic.svelte
Comment (trueb2) [inline src/routes/Automatic.svelte:38]: Please don't use pixels where possible. We want to be using em's so that the iPad, Surface, and laptop browser experiences stay similar. At this point I think that we are going to have a final sweep at the end to go through and fix our px usages.

File: src/routes/Automatic.svelte
Comment (trueb2) [inline src/routes/Automatic.svelte]: dead code :: code smell. I know this about to change so no biggie

File: src/routes/Manual.svelte
Comment (trueb2) [inline src/routes/Manual.svelte:134]: If we could pull out the block math and command building, this file could start to be really clean. Pulling this apart further would be overkill. I think this is just about the right amount of complexity until there are further needs.

**Review by trueb2 — COMMENTED** [review]
I left some comments. None of them are blockers. I like these changes. Since there are more to come in a similar vein for the Automatic page, I will leave it up to you. Do you want me to merge this, while you handle cleanup in a separate PR or would you rather put a bow on these changes before merging?

File: src/routes/Manual.svelte
Comment (rschloen) [reply src/routes/Manual.svelte:272]: That was just a temporary 'hacky' way to fix the spacing so the timing column lined up with messages at the top of the column on the left when I changed the layout so the hexagons don't get cutoff at the bottom. Will likely be replaced when I go through and finalize the layout for everything.

**Review by trueb2 — APPROVED** [review]

**Pull request merged**
